backend/LcmCrdt.Tests/LexboxApiTests.cs

```diff
@@ -367,13 +425,19 @@ public async Task UpdateSensePartOfSpeech()
             entry.Senses[0].Id,
             _api.CreateUpdateBuilder<Sense>()
                 .Set(e => e.PartOfSpeech, "updated")
+                .Set(e => e.PartOfSpeechId, partOfSpeechId)
                 .Build());
         updatedSense.PartOfSpeech.Should().Be("updated");
+        updatedSense.PartOfSpeechId.Should().Be(partOfSpeechId);
     }
 
     [Fact]
     public async Task UpdateSenseSemanticDomain()
     {
+        var newDomainId = Guid.NewGuid();
+        await DataModel.AddChange(Guid.NewGuid(), new CreateSemanticDomainChange(newDomainId, new MultiString() { { "en", "test" } }, "updated"));
```

Comment:
Similar concern here as with the part of speech test: if we're interfacing with FieldWorks, we should have the same behavior that FieldWorks does, and not allow semantic domains with GUIDs that don't appear in the master list. (Though currently we don't have any predefined semantic domains, so as with the part of speech test, my opinion is that this one should pass for now, but eventually should be changed to expect an API error when a custom GUID is sent).